src/python/grpcio/grpc/__init__.py

```diff
@@ -1963,6 +1963,7 @@ class Compression(enum.IntEnum):
     'StatusCode',
     'Status',
     'RpcError',
+    'AioRpcError',
```

Comment:
I think we shouldn't touch our API surface now. `AioRpcError` should belong to the init file of experimental/aio instead of the main init file.